src/virtual-keyboard.component.ts

```diff
@@ -67,7 +67,7 @@ import { KeyPressInterface } from './key-press.interface';
 })
 
 export class VirtualKeyboardComponent implements OnInit, OnDestroy {
-  @ViewChild('keyboardInput') keyboardInput: ElementRef;
+  @ViewChild('keyboardInput', {static: true}) keyboardInput: ElementRef;
```

Comment:
```suggestion
  @ViewChild('keyboardInput', { static: true }) keyboardInput: ElementRef;
```

Not major - just personal style that I have used in other Angular projects and I think that this same is used quite globally in our applications.

Reply:
I agree. I had my autoformat disabled because it did some funky stuff with app.component.html, I should fix that.